portal-test/src/com/liferay/portal/kernel/service/test/PortalRegisterTestUtil.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.portal.service.test;
+package com.liferay.portal.kernel.service.test;
```

Comment:
@ChiZelin @dantewang 
This class is already deprecated, why we should move it to portal-test?

Tina.